ci_framework/roles/edpm_deploy_baremetal/tasks/main.yml

```diff
@@ -192,9 +192,9 @@
       oc patch openstackdataplanenodeset openstack-edpm-ipam
       -n {{ cifmw_install_yamls_defaults['NAMESPACE'] }}
       --type json
-      -p '[{"op": "replace",
+      -p '[{"op": "add",
             "path": "/spec/nodeTemplate/ansible/ansibleVars/edpm_container_registry_insecure_registries",
-            "value": "['{{ content_provider_registry_ip }}:5001']"}]'
+            "value": [{{ content_provider_registry_ip }}:5001]}]'
```

Comment:
@raukadah If `}]'` at this line where placed at the next line I'd be really really easier to follow the closing of each quote and bracket. It's fine, just a comment.